src/main/kotlin/graph/Graph.kt

```diff
@@ -60,5 +60,10 @@ abstract class Graph<V> {
         weight: Int,
     )
 
+    abstract fun getShortestPath(
+        start: V,
+        end: V,
+    ): MutableList<V>?
```

Comment:
I think we should use MutableList? of Ints here, i.e a list of indices. It would be much better for our purposes.

Reply:
realno, typanul chot